docs/01_introduction/README.md

```diff
@@ -214,8 +214,9 @@ specify changes that should be made to one or more
 [shell](https://en.wikipedia.org/wiki/Shell_(computing))-agnostic way. A module file
 is usually written in either [Tcl](https://en.wikipedia.org/wiki/Tcl) or
 [Lua](https://en.wikipedia.org/wiki/Lua_(programming_language)) syntax,
-and specifies for which environment variables the value should be updated, and how (append,
-prepend, (re)define, undefine, etc.).
+and specifies for which environment variables should be updated, and how (append,
+prepend, (re)define, undefine, etc.) upon loading the environment module. Upon unloading
+the environment module the shell environment is returned to its previous state.
```

Comment:
"Unloading the environment module results in restoring the shell environment to its previous state."

Reply:
"Unloading the environment module will restore the shell environment to its previous state."? "Results" and "restoring" kinda clobber each other there.

Reply:
OK